server/controllers/games.js

```diff
@@ -0,0 +1,73 @@
+const Game = require('../models').Game;
+
+
+module.exports= {
+    create(req, res) {
+        return Game
+            .create({
+                game_number: req.body.game_number,
+                sr_change: req.body.sr_change,
```

Comment:
sr change can be calculated from the data. I don't think it needs to be stored in the database